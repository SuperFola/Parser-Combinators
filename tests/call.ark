(func a b)
(
func
(if 1 2 3)
"hello"
)