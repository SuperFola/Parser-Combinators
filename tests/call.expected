( Symbol:func Symbol:a Symbol:b )
( Symbol:func ( Keyword:if Number:1 Number:2 Number:3 ) String:hello )